Meadowlark-js/.vscode/settings.json

```diff
@@ -98,5 +98,9 @@
       "markdown",
       "batch"
     ]
-  }
+  },
+  "jest.autoRun": {
+    "watch": false
+  },
+  "jest.jestCommandLine": "\"./node_modules/.bin/jest\" --runInBand",
```

Comment:
This allows the Jest extension to run all tests at the same time without having conflicts